### ISSUE TITLE
Update to latest uap-core

### ISSUE THREAD

**Issue by asalant** [ua-parser/uap-python#54]
@mattrobenolt - unfortunately I missed an issue in `uap-core` that left my fix for Pingdom bot detection incomplete. https://github.com/ua-parser/uap-python/pull/53 includes that fix. I'd appreciate another patch release of `uap-python`. 

Thanks!

**Comment by asalant** [comment]
@mattrobenolt another ping up this release request. :pray:

**Comment by mattrobenolt** [comment]
sigh part of the way this project works is it's possible for you to link to whatever files you want as a source of rules. A bump here isn't a requirement. You can set the `UA_PARSER_YAML` environment variable and point it to whatever file you want.

https://github.com/ua-parser/uap-python/blob/master/ua_parser/user_agent_parser.py#L449-L450

There is a reason why this isn't 100% in sync all the time and I don't bump versions on every single commit upstream.

**Comment by asalant** [comment]
Ah, that's super helpful. Would be a good addition to the README to help reduce nagging from folks like me. 

I'm fine with going that route for now. It does leave the "fix" that I contributed in a halfway complete state but it will get resolved the next time you sync with aup-core.

**Comment by mattrobenolt** [comment]
So here's the current dilemma updating to latest master ua-parser/uap-core@878312901668097820672351015ecb10796bdcb4

```
....F.....
======================================================================
FAIL: testBrowserscopeStringsOS (__main__.ParseTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "ua_parser/user_agent_parser_test.py", line 48, in testBrowserscopeStringsOS
    TEST_RESOURCES_DIR, 'tests/test_os.yaml'))
  File "ua_parser/user_agent_parser_test.py", line 191, in runOSTestsFromYAML
    result['patch_minor']))
AssertionError: UA: Mozilla/5.0 (Windows U; Win NT 5.0; en-US; rv:1.8.0.2) Gecko/20060308 Firefox/1.5.0.2
 expected<Windows NT None None None> != actual<Windows $1 None None None>

----------------------------------------------------------------------
Ran 10 tests in 44.042s

FAILED (failures=1)
```

So will need to investigate why this test now fails. :(

**Comment by meagerman** [comment]
I found a relatively painless way to integrate the latest yaml into my build. I just downloaded the rules and committed it in a static directory that gets deployed with my project, and referenced it using a relative path like this before importing `ua_parser`:

```os.environ['UA_PARSER_YAML'] = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'static', 'user_agents.yaml')```

Works well enough.

**Comment by panther7** [comment]
Hello, please update to latest (June 2017 or newer). Thanks.

**Comment by ondras** [comment]
@mattrobenolt please, what is the status with this? We are currently unable to use uap-python from pip due to old regex definitions...

**Comment by ondras** [comment]
The fix is ready to be merged in https://github.com/ua-parser/uap-python/pull/59 .

**Comment by mattrobenolt** [comment]
Fixed by 0.8.0 https://pypi.org/project/ua-parser/0.8.0/
